vendor/assets/javascripts/jquery.jnotify.js

```diff
@@ -160,7 +160,8 @@
 
 		// remove the notification
 		this.remove = function (callback){
-			var $msg = $note.find("." + options.classMessage), $parent = $msg.parent();
+
+			var $msg = $note.find("." + options.classMessage.replace(" ",",")), $parent = $msg.parent();
```

Comment:
Lets get this pr in first, then we can change the alerts and remove this vendor change.